### ISSUE TITLE
Confirm message is not accessible when it is opened.

### ISSUE THREAD

**Issue by jhkimnew** [Microsoft/IIS.WebManager#368]
I showed the symptom to Peter, which we found during the code review of the recent PR. 
He asked to create a new issue.

![image](https://user-images.githubusercontent.com/20650163/57880221-11167500-77d3-11e9-8aa8-27f688dd449f.png)

**Comment by shirhatti** [comment]
@jhkimnew Can you verify this still exists?

**Comment by jhkimnew** [comment]
@shhsu and I discussed on this issue and we decided to remove the popup error message because we don't use it when we stop/start from the General feature of each web site.

So, there are two things to change in order to resolve this issue.

1. Considering we don't show the popup message in other user scenario, we should remove the popup message UI. 

2. Loading wheel is not shown when I tried to stop/start site from the Web Sites feature page.
